src/services/ReportHub/ReportHub.BLL/Extensions/Utils/GrpcClientFactory.cs

```diff
@@ -0,0 +1,21 @@
+using Grpc.Net.Client;
+using ProtoBuf.Grpc.Client;
+
+namespace ReportHub.BLL.Extensions.Utils
+{
+    public class GrpcClientFactory
```

Comment:
Where are you using this class? I didn't find any references to it

Reply:
I removed it